libs/full/runtime_distributed/tests/regressions/dijkstra_termination_disconnected_locality_7474.cpp

```
@@ -0,0 +1,78 @@
+#include <vector>
+
+// Per-locality flag, flipped the first time this locality is asked to
+// participate in Dijkstra termination detection (i.e. its color state is
+// consulted via dijkstra_termination_action).
+std::atomic<bool> consulted(false);
+
+void mark_consulted()
+{
+    consulted.store(true);
+}
+HPX_PLAIN_ACTION(mark_consulted, mark_consulted_action)
```

Cppcheck (2.21.0)

[error] There is an unknown macro here somewhere. Configuration is required. If HPX_PLAIN_ACTION is a macro then please configure it.

(unknownMacro)